docs/config/README.md

```diff
@@ -71,6 +73,50 @@ os.setenv('STARSHIP_CACHE', 'C:\\Users\\user\\AppData\\Local\\Temp')
 
 By convention, most modules have a prefix of default terminal color (e.g. `via` in "nodejs") and an empty space as a suffix.
 
+### Strings
+
+In TOML syntax, [text values](https://toml.io/en/v1.0.0#string) are declared with `'`, `"`, `'''`, or `"""`.
+
+The following Starship syntax symbols have special usage in a format string and must be escaped to display as that character: `$ [ ] ( )`.
+
+| Symbol | Type                      | Notes         |
+| ------ | ------------------------- | ------------- |
+| `'`    | literal string            | recommended   |
+| `"`    | string                    | more escaping |
+| `'''`  | multi-line literal string |               |
+| `"""`  | multi-line string         |               |
+
+For example:
+
+```toml
+# literal string (recommended)
+format = '☺\☻ '
+
+# regular string
+format = "☺\\☻ "
+
+# escaping Starship symbols
+format = '\[\$\] '
+```
+
+When using line breaks, multi-line declarations can be used.
+For example, if you want to print a `$` symbol on a new line, the following values for `format` are equivalent:
+
+```toml
+# with literal string (recommended)
+format = '''
+
+\$'''
```

Comment:
I feel like `"""` should be preferred over `'''`.  Using a construction like in the [`$all` explanation](https://starship.rs/config/#default-prompt-format) seems to be somewhat popular among users.

Reply:
What do you think of this?

I dropped the recommendation, and described the possibility to use such escaped newlines.